core/commands/stat.go

```diff
@@ -116,14 +150,14 @@ var statBwCmd = &cmds.Command{
 					totals := nd.Reporter.GetBandwidthTotals()
 					out <- &totals
 				}
-				if !doPoll {
-					return
-				}
-				select {
```

Comment:
I think it is what will happen over here.

If `doPoll` is false it will go straight to the `return statement in line 160.